source/process/guest-article.rst

```diff
@@ -0,0 +1,21 @@
+Guest Articles for Mattermost Apps and Services
+===============================================================
+
+This document outlines some of the procedures in developing guest articles. Mattermost, Inc. is excited to bring attention to guest company creators and their experiences with using Mattermost.
```

Comment:
Is it `some of the procedures` or `the procedures`? 

If it's only some, where's the rest?